src/badge/badge.scss

```diff
@@ -11,14 +11,17 @@ $iui-badge-background-primary: $iui-color-foreground-primary--dark;
 $iui-badge-text-color: rgba($iui-color-foreground-body--light, $iui-opacity-3--light);
 
 @mixin iui-badge {
+  --_iui-badge-background-color: #{$iui-badge-background-default};
+  --_iui-badge-text-color: #{$iui-badge-text-color};
```

Comment:
I think these variables should be considered public and not have an underscore, as iTwin UI CSS could be (or is? Blazor?) consumed by more than just our React project.

Reply:
Removed `_` from the background-color variable. The text color variable is not used externally